src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/UserCreationRequestValidator.java

```diff
@@ -4,26 +4,31 @@
 
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
 import org.springframework.util.CollectionUtils;
 import uk.gov.hmcts.reform.professionalapi.controller.request.InvalidRequest;
 
+@Component
 @Slf4j
 public class UserCreationRequestValidator {
 
-    @Value("${logging-component-name}")
-    protected static String loggingComponentName;
+    private static String loggingComponentName;
 
     private UserCreationRequestValidator() {
     }
 
-
     public static List<String> validateRoles(List<String> roles) {
 
         if (CollectionUtils.isEmpty(roles)) {
-            log.error("{}:: No user role(s) provided",loggingComponentName);
+            log.error("{}:: No user role(s) provided", loggingComponentName);
```

Comment:
better remove logger.anyway throwing error right. so no need add extra code for logger

Reply:
Addressed in this commit: https://github.com/hmcts/rd-professional-api/pull/649/commits/8d6014387fe7f49d589bc62bb8104ce838d8afbb